.github/workflows/matrix.js

```diff
@@ -3,6 +3,7 @@
 // See https://github.com/vlsi/github-actions-random-matrix
 let fs = require('fs');
 let os = require('os');
+const { RNG } = require('./rng');
```

Comment:
```suggestion
const { rng } = require('./rng');
```

Reply:
Why? That would break the code as the name of the export in `./rng` is `RNG` (all caps).

Reply:
Is there a reason to use all-caps naming? The convention is lowercase like `fs`, `crypto` and so on

Reply:
The node convention of using lowercase is for importing entire built-in modules.

This is a named export of a constant initialized in the `./rng`, hence the uppercase name in the definition and import.

Reply:
Random number generator is mutable, and it is not constant, so it goes lowercase

Reply:
No, it is a constant. That's the verbiage used in JavaScript to refer to a value initialized once. The value is defined once in `./rng` and that value is being imported and referenced. The internals being mutable are irrelevant.

Reply:
See https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Reply:
@sehrope , here's the relevant Google guide for JavaScript: https://google.github.io/styleguide/jsguide.html#naming-constant-names

> For example, if any of that instance's observable state can change, it is almost certainly not a constant. Merely intending to never mutate the object is generally not enough

The relevant sample is `const /** Set<string> */ mutableCollection = new Set();`, so it justifies that the exported random number generator should better named with lower case.

Reply:
Please stop wasting time on these meaningless comments. You're linking to a Java style guide for the naming of an internal JavaScript variable.

I think your time would be better spent on more meaningful work in the repo.

Reply:
Please read the link again. https://google.github.io/styleguide/jsguide.html#naming-constant-names describes `Google JavaScript Style Guide`